lib/internal/loader/DefaultResolve.js

```diff
@@ -39,14 +43,22 @@ function search(target, base) {
   }
 }
 
-const extensionFormatMap = {
+const extensionFormatStd = {
```

Comment:
Perhaps this should be `extensionFormatCjs`, as this default may change over time.

Reply:
quick ping re naming it CJS